connectors/services/job_scheduling.py

```diff
@@ -24,7 +24,7 @@
 from connectors.es.index import DocumentNotFoundError
 from connectors.logger import logger
 from connectors.services.base import BaseService
-from connectors.source import get_source_klass_dict
+from connectors.source import get_source_klass, get_source_list
```

Comment:
Don't feel it's necessary to introduce these 2 methods, which just return dict keys and a dict.

Reply:
Sorry I did not get your comment @wangch079, what do you mean?

Reply:
oh I see ! you're right :)